sinks/api/metricdecoder.go

```diff
@@ -121,24 +123,36 @@ func (self *defaultDecoder) getContainerMetrics(container *source_api.Container,
 				} else {
 					startTime = stat.Timestamp
 				}
-				points := supported.GetValue(&container.Spec, stat)
-				for _, point := range points {
+				seriesName := supported.Name
+				if supported.Units.String() != "" {
+					seriesName = fmt.Sprintf("%s_%s", seriesName, supported.Units.String())
+				}
+				if supported.Type.String() != "" {
+					seriesName = fmt.Sprintf("%s_%s", seriesName, supported.Type.String())
+				}
+				internalPoints := supported.GetValue(&container.Spec, stat)
+				points := []Point{}
+				for _, internalPoint := range internalPoints {
 					labels := copyLabels(labels)
-					for name, value := range point.labels {
+					for name, value := range internalPoint.labels {
 						labels[name] = value
```

Comment:
These labels might not show up as part of LabelKeys. Why not include all metric labels by default as part of labelkeys?